File: Server-Spring-Java/src/main/java/pyo/quizgame/domain/ServerInfo.java
package pyo.quizgame.domain;

import lombok.Data;

import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.OneToOne;

@Data
@Entity
public class ServerInfo {

    @Id
    private Long id;
    private Long totalNumberOfQuiz;
    private Long totalNumberOfUser;
    @OneToOne
    private FrontUserInfo topQuizPlayer;
    @OneToOne
    private FrontUserInfo badQuizPlayer;
    @OneToOne
    private FrontUserInfo bestQuizPlayer;
}

File: Front-Android-Kotlin/app/src/main/java/com/pyo/quizfrontapp/data/ServerInfo.kt
package com.pyo.quizfrontapp.data

data class ServerInfo(
    var id:Long,
    var totalNumberOfQuiz:Long,
    var totalNumberOfUser:Long,
    var topQuizPlayer:FrontUserInfo,
    var badQuizPlayer:FrontUserInfo,
    var bestQuizPlayer:FrontUserInfo,

)
